Internal/Debug App/Project.swift

```diff
@@ -14,7 +14,7 @@ enum AppSettings {
 
     static let settingsDictionary = SettingsDictionary()
         .merging(BaseSettings.settingsDictionary)
-        .merging(["CODE_SIGN_IDENTITY": .string("Apple Development: DX Primer (8B5K7AGMS8)")])
+        .merging(["CODE_SIGN_IDENTITY": .string("Apple Development: MOHAMMED TARIK ALADIN TALEB (P3K69N82DT)")])
```

Comment:
Is that required to have my name here @semirp ? :p

Reply:
yes 🗡️